content/ordering.ts

```diff
@@ -123,8 +123,8 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
   ],
   gold: [
     {
-      name: 'Introductory Number Theory',
-      items: ['divisibility', 'modular'],
+      name: 'Math for Competitive Programming',
```

Comment:
```suggestion
      name: 'Introductory Number Sense',
```
doesn't sound as cringe as "Math for CP"